frontend-service/src/app/js/components/Landing.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
```

Comment:
Preferably, just name all your javascript files with the `.js` extension.

Reply:
Noted